pcap/pcap.h

```diff
@@ -866,7 +866,9 @@ PCAP_API const char *pcap_lib_version(void);
 
   PCAP_API int pcap_start_oem(char* err_str, int flags);
 
-  PCAP_API PAirpcapHandle pcap_get_airpcap_handle(pcap_t *p);
+  PCAP_AVAILABLE_1_10
```

Comment:
This API has always been available in the versions of tcpdump.org's libpcap for Windows; as it's now a deprecated API, I wouldn't bother decorating it with a `PCAP_AVAILABLE_` tag.

I'd also leave the `*` styling as is.